classes/02class/exercises/c02-network05/frdvo/README.md

```diff
@@ -0,0 +1,42 @@
+# c02-network05
+
+## Commands Execution Output
+
+- Commands for creating the internet gateway:
+```
+#Create Internet Gateway
+aws ec2 create-internet-gateway
+
+#Tag Internet Gateway
+aws ec2 create-tags --resources igw-0116304685cc6f00b --tags Key=Name,Value=devopsacademy-igw
+
+#Attach Internet Gateway
+aws ec2 attach-internet-gateway --internet-gateway-id igw-0116304685cc6f00b --vpc-id vpc-0a2b7db4956438d22
```

Comment:
can you run `aws ec2 describe-internet-gateways --internet-gateway-ids $IGW` after attaching and include the output here?

Reply:
Easy